packages/yoastseo/src/scoring/assessments/seo/TextLengthAssessment.js

```diff
@@ -192,7 +192,8 @@ export default class TextLengthAssessment extends Assessment {
 	}
 
 	/**
-	 * Returns the score and the appropriate feedback string based on the current word count.
+	 * Returns the score and the appropriate feedback string based on the current word count for every type of content,
+	 * apart from taxonomies (in WordPress) and collections (in Shopify).
```

Comment:
Actually this also include taxonomies (in WordPress) and collections (in Shopify), because `calculateTaxonomyResult` is also called in this function `calcultaeResult` :)

Reply:
Hahah very true! Will remove that part of the comment :)